grabbit/tests/specs/test.json

```diff
@@ -1,4 +1,8 @@
 {
+  "index" : {
+    "exclude" : ["derivatives"],
```

Comment:
Example feels a bit weird because there's no reason to check both include and exclude... if you define `include`, that implies that everything not in `include` is to be excluded, and vice versa for `exclude`. So we probably need at least one additional .json spec, so we can test both options. Also, maybe add more than one element to the list, to make sure it behaves properly in that case.

Reply:
Yeah I agree.